backend/internal/storage/postgres/schema/review.go

```diff
@@ -0,0 +1,103 @@
+package user
+
+import (
+	"context"
+	"platnm/internal/models"
+
+	"time"
+
+	"github.com/jackc/pgx/v5/pgxpool"
+)
+
+type ReviewRepository struct {
+	db *pgxpool.Pool
+}
+
+func (r *ReviewRepository) GetReviews(ctx context.Context) ([]*models.Review, error) {
+	rows, err := r.db.Query(context.Background(), "SELECT user_id, media_id, media_type, desc, rating, CreatedAt, UpdatedAt FROM review")
+	if err != nil {
+		print(err.Error(), "from transactions err ")
+		return []*models.Review{}, err
+	}
+	defer rows.Close()
+
+	var reviews []*models.Review
+	for rows.Next() {
+		var review models.Review
+		var userID, mediaID, mediaType, desc, rating *string
```

Comment:
dont think u need these temp vals. can just bind straight to fields of review struct

Reply:
ya see [this](https://pkg.go.dev/github.com/jackc/pgx/v5#example-RowToStructByName) from the pgx docs